src/test/java/com/fastcampus/projectboard/domain/AuditingFieldsReflectionHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.fastcampus.projectboard.domain;
+
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.time.LocalDateTime;
+
+public class AuditingFieldsReflectionHelper {
+
+    private AuditingFieldsReflectionHelper() {
+
+    }
+
+    public static void setAuditingFields(AuditingFields entity, int offset, LocalDateTime localDateTime) {
+        setAuditingFields(entity, offset, localDateTime, 3);
+    }
+
+    public static void setAuditingFields(AuditingFields entity, int offset, LocalDateTime localDateTime, int modifiedAddMinute) {
+        ReflectionTestUtils.setField(entity, "createdAt", localDateTime.plusMinutes(offset));
+        ReflectionTestUtils.setField(entity, "createdBy", "createdBy" + offset);
+        ReflectionTestUtils.setField(entity, "modifiedAt", localDateTime.plusMinutes(modifiedAddMinute + offset));
+        ReflectionTestUtils.setField(entity, "modifiedBy", "modifiedBy" + offset);
+    }
```

Comment:
저는 지난 리뷰에서 `offset`이 필요없어 보인다는 말씀을 아래와 비슷한 아이디어에서 말씀드린 거였어요.

```suggestion
    public static void setAuditingFields(AuditingFields entity, LocalDateTime localDateTime, int modifiedAddMinute) {
        ReflectionTestUtils.setField(entity, "createdAt", localDateTime);
        ReflectionTestUtils.setField(entity, "createdBy", "createdBy" + localDateTime.getMinute());
        ReflectionTestUtils.setField(entity, "modifiedAt", localDateTime.plusMinutes(modifiedAddMinute));
        ReflectionTestUtils.setField(entity, "modifiedBy", "modifiedBy" + localDateTime.getMinute());
    }
```

Reply:
아 offset으로 사용했던 int값을 localDateTime에서 뽑아낼 수 있겠군요 감사합니다